controllers/clusteroperator_controller.go

```diff
@@ -180,6 +191,27 @@ func getResources(infra *configv1.Infrastructure) []client.Object {
 	return nil
 }
 
+func (r *CloudOperatorReconciler) composeConfig(infra *configv1.Infrastructure) (operatorConfig, error) {
+	config := operatorConfig{}
+
+	images, err := getImagesFromJSONFile(r.ImagesFile)
+	if err != nil {
+		klog.Errorf("Unable to decode images file from location %s", r.ImagesFile, err)
+		return config, err
+	}
+
+	platform, err := getProviderFromInfrastructure(infra)
+	if err != nil {
+		klog.Errorf("Unable to decode images file from location %s", r.ImagesFile, err)
+		return config, err
+	}
```

Comment:
We already have platform extracted in the parent scope, could just pass it in here rather than getting it again. Less error handling/complexity :)

Reply:
Addressed this one. All should be fixed now, PTAL.